__tests__/src/components/SearchPanelNavigation.test.js

```diff
@@ -17,26 +18,28 @@ function createWrapper(props) {
 
 describe('SearchPanelNavigation', () => {
   describe('when searchHits are available', () => {
-    it('renders text with buttons', () => {
+    it('renders text with buttons', async () => {
       const selectAnnotation = jest.fn();
-      const wrapper = createWrapper({
+      const user = userEvent.setup();
+      createWrapper({
         searchHits: [{ annotations: ['1'] }, { annotations: ['2'] }, { annotations: ['3'] }],
         selectAnnotation,
         selectedContentSearchAnnotation: ['2'],
       });
-      expect(wrapper.find('WithStyles(ForwardRef(Typography))').text()).toEqual('pagination');
-      expect(wrapper.find('WithWorkspaceContext(WithPlugins(MiradorMenuButton))[disabled=false]').length).toEqual(2);
-      wrapper.find('WithWorkspaceContext(WithPlugins(MiradorMenuButton))[disabled=false]').first().props().onClick();
+      expect(screen.getByText('pagination')).toBeInTheDocument();
+      expect(screen.getAllByRole('button').length).toEqual(2);
+      await user.click(screen.getByRole('button', { name: 'searchPreviousResult' }));
       expect(selectAnnotation).toHaveBeenCalledWith('1');
-      wrapper.find('WithWorkspaceContext(WithPlugins(MiradorMenuButton))[disabled=false]').last().props().onClick();
+      await user.click(screen.getByRole('button', { name: 'searchNextResult' }));
       expect(selectAnnotation).toHaveBeenCalledWith('3');
```

Comment:
Sorry if I'm missing something but -- selected is 2. We click searchPrevious and get 1. Then we hit searchNext and get 3? Shouldn't we be back on 2?

Reply:
Except we're not actually updating the underlying state, so we never go anywhere 🤷‍♂️